pylintrc

```diff
@@ -101,7 +105,10 @@ ignore-mixin-members=yes
 
 # List of classes names for which member attributes should not be checked
 # (useful for classes with attributes dynamically set).
-ignored-classes=
+ignored-classes=pyoozie.model._OozieArtifact,
+ pyoozie.model._StatusValue, pyoozie.model._OozieArtifact.StatusEnum,
+ pyoozie.model.Coordinator, pyoozie.model.CoordinatorAction,
+ pyoozie.model.Workflow, pyoozie.model.WorkflowAction,
```

Comment:
Listing classes here prevents _some_ of the lint errors around `no-member` etc. -- but not all.
The `generated-members=` list below should also do this, but it doesn't appear to work properly with wildcards (despite the changelog entry promising exactly that).